app/src/main/cpp/skyline/audio/resampler.cpp

```diff
@@ -18,7 +16,7 @@ namespace skyline::audio {
     };
 
     // @fmt:off
-    constexpr std::array<LutEntry, 128> CurveLut0 = {{
+    constexpr std::array<LutEntry, 128> CurveLut0{{
```

Comment:
we should probably make this static, saw a pr about smth similar in yuzu

Reply:
Bro when this project skyline will be able to run switch games? I support you guys but I don't know coding if is there any work which I can do for you guys just message me ☺️ plz reply

Reply:
There is no reason to do so, `static` does not affect this